src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/Endpoints/RazorCustomMessageTarget_CodeActions.cs

```diff
@@ -82,7 +80,7 @@ await foreach (var response in requests.WithCancellation(cancellationToken).Conf
             {
                 foreach (var codeAction in response.Response)
                 {
-                    codeAction.Data = JsonHelpers.TryConvertFromJsonElement(codeAction.Data);
+                    codeAction.Data = JsonHelpers.TryConvertFromJObject(codeAction.Data);
```

Comment:
The razor custom message target stuff sits between the Razor LSP server and a delegated LSP server (either Roslyn or WebTools), so whilst the Razor server is using a specific Json engine, we have no control over what Roslyn or WebTools does. To avoid dual or triple insertions, these methods were added so that we allow delegated servers to give us either Newtonsoft or STJ, and we handle converting them to whichever system the Razor LSP uses, and converting them back as necessary (for code action resolve in this case). This was done in https://github.com/dotnet/razor/pull/10494

This PR changes the "whichever system the Razor LSP uses", so these methods changed in line with that. ie, they used to convert from a delegated server using STJ, to Razor which used Newtonsoft. Now they convert from a delegated server using Newtonsoft, to Razor which uses STJ. If a delegated server uses STJ, then there is no conversion necessary.